data/layout/telescope_positions-North-4SST.ecsv

```diff
@@ -51,8 +46,8 @@
 #       unit: *id001
 #       value: 3.0
 # schema: astropy-2.0
-telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
-SST-01 80.0 80.0 80.0 28.762879811645295 -17.89239016473408 217575.98596326663 3185147.2580149896 2154.75
-SST-02 -80.0 -80.0 -80.0 28.761436183311933 -17.892390153463488 217572.0961101121 3184987.211762427 2154.75
-SST-03 80.0 80.0 80.0 28.762879811645295 -17.890751835265917 217736.03212229832 3185143.368077023 2154.75
-SST-04 -80.0 -80.0 -80.0 28.761436183311933 -17.890751846536514 217732.14227786276 3184983.322002804 2154.75
+telescope_name      pos_x      pos_y      pos_z
+SST-01               80.0       80.0       80.0
+SST-02              -80.0      -80.0      -80.0
```

Comment:
SSTs in the north, neg z.

Reply:
SST files for the North were now removed